deploy/templates/service-ca/service_cert_patch.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: manager-service
+  namespace: system
+  annotations:
+    service.beta.openshift.io/serving-cert-secret-name: devworkspace-webhooks-tls
```

Comment:
What do you mean for using an env var on OS? 

In Kubernetes, we have to create the cert manually, and assign DNS names to it to match our services. Since these TLS certs have to be created beforehand and reused in the webhooks server, the controller deployment needs to know the secret name.

In OpenShift, the service CA operator generates TLS secrets from the annotation on the service, and that gets mounted, but we can't reuse the TLS secret that was generated for the controller since it's only valid for the controller's service. We could read that secret's name from an environment variable, but it would amount to a one-line change in the code.

Reply:
OS - openshift.

> We could read that secret's name from an environment variable, but it would amount to a one-line change in the code.

it's what I meant

```diff
-               secretName = server.WebhookServerTLSSecretName
+               secretName, err = config.GetWebhooksSecretName()
+               if err != nil {
+                       return fmt.Errorf("could not deploy webhooks server: %w", err)
+               }
```
https://github.com/devfile/devworkspace-operator/blob/main/pkg/webhook/create.go#L60

But seems it also requires changing adding kustomize patch for openshift.... So, feel free to just resolve conversation .

Reply:
Implemented in edf54b4a..76ac02c1, though the original suggestion still can't be done, since we use the same cert for both the controller and webhooks server when running in Kubernetes.

Reply:
I love github links to diff in PR. It does not work in 90% cases )
But I found the corresponding PR and it LGTM